benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/resources/BenchFlowTestResource.java

```diff
@@ -212,7 +221,39 @@ public BenchFlowTestModel getBenchFlowTestStatus(@PathParam("username") String u
     BenchFlowTestModel benchFlowTestModel = null;
 
     try {
+
       benchFlowTestModel = testModelDAO.getTestModel(testID);
+
+      // set the URL to the exploration point index
+      // it is done dynamically since we need the server information
+      for (BenchFlowExperimentModel experimentModel : benchFlowTestModel.getExperimentModels()) {
+
+        int pointIndex = experimentModel.getExplorationPointIndex();
+
+        // add url only if exploration space is not empty or null
+
+        MongoCompatibleExplorationSpace explorationSpace =
+            explorationModelDAO.getExplorationSpace(testID);
+
+        String url;
+
+        if (explorationSpace == null || explorationSpace.getSize() == 0) {
```

Comment:
I suggest to move this into an issue since it is not important for the use cases.